sqlmesh/cli/main.py

```diff
@@ -153,6 +154,7 @@ def render(
     execution_time: t.Optional[TimeLike] = None,
     expand: t.Optional[t.Union[bool, t.Iterable[str]]] = None,
     dialect: t.Optional[str] = None,
+    no_pretty_print: bool = False,
```

Comment:
i was confused because i thought you meant sqlglot pretty, but you're keeping this, it's really in the console, show sql.

maybe it should be

--no-format

Reply:
right, true